### ISSUE TITLE
Stable version tag?

### ISSUE THREAD

**Issue by adamv** [RobertAudi/fire#1]
Can we get a tag for a non-HEAD version of this?

Looks like development has slowed somewhat, so perhaps a 0.1.0 version is in order?

**Comment by RobertAudi** [comment]
I you're interested I can add you as a contributor, but I don't have the time to check if fire is stable in the state it is now. I don't know if you have noticed but I created an experimental branch where I rewrote fire from scratch. That branch might be more stable than the HEAD. The thing I don't want to do is just add a stable tag without any prior checks. Unfortunately, I have no more interest in PHP, I became a rubyist a couple month ago and I am not coming back. So once again, if you're interesting in becoming the new maintainer than be my guest, I'll add you as contributer. Otherwise fork the project; that's why I've put it on github in the first place.

I am very sorry to say that as far as I'm concerned, the fire project is dead. Long live fire!
